src/chainlit/socket.py

```diff
@@ -98,31 +98,30 @@ def ask_user_fn(data, timeout):
 
 @socket.on("connection_successful")
 async def connection_successful(sid):
-    session = Session.require(sid)
-    if session.restored:
-        return
-
-    emitter_var.set(ChainlitEmitter(session))
-    loop_var.set(asyncio.get_event_loop())
-
-    if isinstance(session.auth_client, CloudAuthClient) and config.project.database in [
-        "local",
-        "custom",
-    ]:
-        await session.db_client.create_user(session.auth_client.user_infos)
-
-    if config.code.on_chat_start:
-        """Call the on_chat_start function provided by the developer."""
-        await config.code.on_chat_start()
+    try:
```

Comment:
let's do that in another PR and wrap all websocket handlers at once

Reply:
actually I had to do that to debug 🤣